packages/pdf-generator/src/utils/styles.js

```diff
@@ -52,6 +52,9 @@ export default (style = {}) => StyleSheet.create({
         fontStyle: 'italic',
         color: '#6e6b6c'
     },
+    reportNameWrapper: {
+        margin: '30, 0, 15'
```

Comment:
I don't think you can do that in react-css, you gotta write it as
```js
marginTop: 30,
marginRight: 0,
marginBottom: 15
```

Reply:
You actually can, [these docs](https://reactnative.dev/docs/layout-props#margin) link [here](https://developer.mozilla.org/en-US/docs/Web/CSS/margin) for more information, which shows option to provide 3 values for top right bottom; so I'd assume it uses that specification. I have also tested It with exec. reports and it looks correct, for example:

```
reportNameWrapper: {
    margin: '300, 100, 200'
},
```

should produce top 300, left and right 100, botttom 200, and it does:
![margins](https://user-images.githubusercontent.com/8426204/91828631-a9db3900-ec40-11ea-8008-d3d8449bbeb7.png)

If you want it to be clearer I can change it to your variant.

Reply:
I was checking docs in react-pdf where they hadn't this option. If it works as expected, then good to go!